Author: Holger Jaekel <holger.jaekel@gmx.de>
Summary: fix build with poppler >= 25.10
         GooString: remove getLength and use std::string::size instead
URL: https://gitlab.freedesktop.org/poppler/poppler/-/commit/f6dcd06445b6b7795b4c8f654206c5aef3d79100
----

--- a/Applications/Cxx/gdcminfo.cxx
+++ b/Applications/Cxx/gdcminfo.cxx
@@ -297,7 +297,7 @@
       isUnicode = false;
       i = 0;
       }
-    while (i < obj.getString()->getLength())
+    while (i < obj.getString()->size())
       {
       if (isUnicode)
         {

--- a/Applications/Cxx/gdcmpdf.cxx
+++ b/Applications/Cxx/gdcmpdf.cxx
@@ -148,7 +148,7 @@
       isUnicode = false;
       i = 0;
       }
-    while (i < obj.getString()->getLength())
+    while (i < obj.getString()->size())
       {
       if (isUnicode)
         {
